res/openvpn-gui-res-it.rc

```diff
@@ -2,6 +2,7 @@
  *  OpenVPN-GUI -- A Windows GUI for OpenVPN.
  *
  *  Copyright (C) 2004 Mathias Sundman <mathias@nilings.se>
+ *                2009 Heiko Hund <heikoh@users.sf.net>
```

Comment:
As as explained last time against your previous PR (#510), do not add or remove copyright of others unless it is your own name. Arbitrarily copying such lines from another files is not appropriate.